src/plugins/api/BioSignalMLAPI/CMakeLists.txt

```diff
@@ -0,0 +1,95 @@
+PROJECT(BioSignalMLAPIPlugin)
+
+MACRO(RETRIEVE_BioSignalML_API_SETTINGS)
+    IF(WIN32)
+        if(CMAKE_BUILD_TYPE MATCHES Debug)
+            set(DEBUG_POSTFIX "_d")
+        endif()
```

Comment:
Consistency: use capital letters, e.g. ```IF(...)``` rather than ```if(...)```. I would also use ```IF("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")``` rather than ```if(CMAKE_BUILD_TYPE MATCHES Debug)```.

Minor: I would put an empty line after the ```IF()``` clause.